html_css_proj/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <style>
+    #navbar {
+      align-items:left;
+      position:fixed
+    }
+    @media only screen and (max-width: 800px) {
+  [class*="col-"] {
+    width: 100%;
+  }
+  </style>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+    <main id="main-doc">
+      <nav id=navbar>
+        <header>
+          Coding
+         </header>
+         <a class="nav-link" href="#introduction">Introduction</a>
+          <a class="nav-link" href="#what_you_should_already_know">what you should already know</a>
+          <a class="nav-link" href="#javascript_and_java">javascript and java</a>
+          <a class="nav-link" href="#hello_world">hello world</a>
+          <a class="nav-link" href="#variables">variables</a>
+      </nav>
+      <section id="introduction" class="main-section">
+        <header id="introduction">
+          Introduction
+        </header>
+        <p>
+          <code>$(element).css({backgroundColor:'transparent'})</code>
+        </p>
+        <p>
+          
+        </p>
+      </section>      
+      <section id="what_you_should_already_know" class="main-section">
```

Comment:
[REQUIRED] Please, avoid using "id" and "class" on the same HTML tag. They work for the same purpose, but "class" can be repeated and used in other tags, while id cannot.